surya/tables.py

```diff
@@ -39,6 +39,19 @@ def sort_bboxes(bboxes, tolerance=1):
     return sorted_page_blocks
 
 
+def pad_to_batch_size(tensor: torch.Tensor, batch_size: int) -> torch.Tensor:
+    current_batch_size = tensor.shape[0]
+    if current_batch_size >= batch_size:
+        return tensor
+
+    pad_size = batch_size - current_batch_size
+    repeats = (pad_size + current_batch_size - 1) // current_batch_size
```

Comment:
Why is this not zero padding?

Reply:
That should work if we were just padding the image tensor but we've also got `batch_bbox_counts` which appears to be constructed in a specific way and breaks the model if it's just zero padded.
https://github.com/VikParuchuri/surya/blob/c33bba493ed3e4926e05f4332770ad51920a642d/surya/model/table_rec/processor.py#L246-L253. So it seemed simpler to do repeat padding. What do you think?